libs/prometheus-operator/config.jsonnet

```diff
@@ -1,12 +1,16 @@
 local config = import 'jsonnet/config.jsonnet';
 
 local versions = [
-    {output: '0.57', version: '0.57.0'},
-    {output: '0.58', version: '0.58.0'},
-    {output: '0.59', version: '0.59.2'},
-    {output: '0.60', version: '0.60.0'},
-    {output: '0.61', version: '0.61.1'},
-    {output: '0.62', version: '0.62.0'},
+  { output: '0.57', version: '0.57.0' },
+  { output: '0.58', version: '0.58.0' },
+  { output: '0.59', version: '0.59.2' },
+  { output: '0.60', version: '0.60.0' },
+  { output: '0.61', version: '0.61.1' },
```

Comment:
Is there any support level for older versions? I'd suggest to just keep the ones that are getting patches.

Reply:
Not sure about support level, as far as I can tell only the latest minor gets patches. Probably a good idea to keep a few versions back?

Reply:
I'd say in this case let's limit it to 10.

Removing it here won't affect users importing it as it's still available with the commit hashed pinned in the jsonnet-bundler lock file